### PR TITLE
Add adi10hero to the gdsc map

### LINKED ISSUE #24: Add DSC VESIT Lead's Data
**Is your feature request related to a problem? Please describe.**
Add DSC vesit lead (me) data

**Additional context**
Nice templates for bugs and features :)

### PR DESCRIPTION
**Pull request by ADI10HERO**
Fixes #24 

## Proposed Changes

- Added photo
- Added json data
- Fixed indentation

### CODE REVIEW

File: src/components/json/worldData.js [hunk 1]
```diff
@@ -186,22 +187,38 @@ const data = [
...
+		"twitter": "https://twitter.com/chrisft25",
+		"portfolio": "https://chrisfuentes.me"
+	},
+	{
+		"longitude": 72.877655,
+		"latitude": 19.075983,
+		"batch": 2020,
+		"photo": adi10hero,
+		"region": "Asia",
+		"skills": ["App Dev", "Web Dev", "UI/UX", "Cloud", "Machine Learning"],
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:213]:
```suggestion
		"skills": ["App Dev", "Web Dev", "UI/UX", "Cloud", "AI/ML"],
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
Hey @ADI10HERO!

You should only use the skill types mentioned in the [contributing guidelines](https://github.com/abhinavsri360/GDSC-Lead-Map/blob/main/CONTRIBUTING.md).

**Comment by ADI10HERO** [comment]
Oh, damn @aravindvnair99  Actually I copied the list from somewhere in the JSON, so please correct that as well.
Sorry I did not check the list as I thought that'll be the correct one as it's in the data JSON already

**Comment by aravindvnair99** [comment]
> Oh, damn @aravindvnair99 Actually I copied the list from somewhere in the JSON, so please correct that as well.
> Sorry I did not check the list as I thought that'll be the correct one as it's in the data JSON already

@ADI10HERO Ah yes, thanks for pointing that out.

https://github.com/abhinavsri360/GDSC-Lead-Map/blob/363021527b40b7e9514b7833f9d770672950c202/src/components/json/worldData.js#L101

**Review by abhinavsri360 — APPROVED** [review]
Thanks for adding yourself!

**Comment by abhinavsri360** [comment]
Closes #24

**Review by aravindvnair99 — APPROVED** [review]

**Pull request merged**